pkg/api/defaults-apiserver.go

```diff
@@ -105,6 +104,10 @@ func (cs *ContainerService) setAPIServerConfig() {
 		defaultAPIServerConfig["--service-account-signing-key-file"] = "/etc/kubernetes/certs/apiserver.key"
 	}
 
+	if !common.IsKubernetesVersionGe(o.OrchestratorVersion, "1.24.0-alpha") {
```

Comment:
Let's change this to `1.24.0-alpha.0`

Reply:
Also cc @alexeldeib in case AKS needs to update its kubelet configuration surface area.

Reply:
as discussed moved this to < 1.20